README.md

```diff
@@ -0,0 +1,25 @@
+# Smaily Drupal module
+
+Smaily email marketing and automation extension plugin for Drupal.
```

Comment:
"extension plugin" mean the same thing, but they do not mean anything in Drupal's context. Drupal uses "module" instead.